Dockerfile

```diff
@@ -1,23 +1,20 @@
-FROM jupyter/scipy-notebook@sha256:4825556416ec7dcf4df73585a71595d29a274ef7d6d7c97267606661f3466952
+FROM jupyter/base-notebook
```

Comment:
Before this is merged can you use a fixed tag from https://hub.docker.com/r/jupyter/base-notebook/tags/